src/Components/LandingPage.js

```diff
@@ -39,26 +39,51 @@ class LandingPage extends Component {
                     </MDBCol>
                 </MDBRow>
 
-                <MDBRow className="landingpage-section">
-                    <MDBCol size="5">
+                <MDBRow className="section-margin">
+                    <MDBCol size="7">
                         <h2 id="section-2-h2">In a new city? Explore it with fellow students!</h2>
                     </MDBCol>
+                    <MDBCol size="5">
+                        <img src="section-2-image.jpg" className="img-fluid" id="section-2-img"></img>
+                    </MDBCol>
+                </MDBRow>
+
+                <MDBRow className="section-margin">
+                    <MDBCol size="12">
+                        <EventCard title="Rockclimbing at Phil's" description="No one has made this event yet...It could be you!" sample={true} location="landing-page"/>
+                    </MDBCol>
+                </MDBRow>
+
+                <MDBRow className="section-margin">
                     <MDBCol size="7">
-                        <img src="section-2-image.jpg" id="section-2-img" className="img-fluid"></img>
+                        <img src="section-3-image.jpg" className="img-fluid" id="section-3-img"></img>
+                    </MDBCol>
+                    <MDBCol size="5">
```

Comment:
Can we try making a constant for certain column and row sizes? Will be easier to tweak if they're all the same. 
Not sure if Jackson wants to propose a better react way to do it but that would be my preference @zhengwin .

Reply:
Can you elaborate on what you mean by a constant?

Reply:
Yeah. declare `const ROW_SIZE = 5` at the top. Or something like that if it's possible??

Reply:
did something like this with `textColSize`, `imgColSize`, and `cardColSize`, lmk what you think